src/elements/content-sidebar/versions/VersionsItem.js

```diff
@@ -118,27 +131,28 @@ const VersionsItem = ({
                         {!!size && <span className="bcs-VersionsItem-size">{versionSize}</span>}
                     </div>
 
-                    {isLimited && (
+                    {isLimited && hasActions && (
```

Comment:
Is this additional check for `hasActions` necessary? Would we want to show the message even if there are no actions?

Reply:
Good question, but no. We only want to show the message if the user would have had actions otherwise. I've validated this with product.

Reply:
👍